doc/sphinx/proofs/writing-proofs/inductive.rst

```diff
@@ -713,34 +709,33 @@ and an explanation of the underlying technique.
    where :n:`@ident` is the identifier for the last introduced hypothesis.
 
 .. tacv:: inversion_clear @ident
-   :name: inversion_clear
 
-   This behaves as :n:`inversion` and then erases :n:`@ident` from the context.
+   Behaves like :n:`inversion` and then erases :n:`@ident` from the context.
 
-.. tacv:: inversion @ident as @or_and_intropattern_loc
+.. tacv:: inversion @ident @as_or_and_ipat
 
-   This generally behaves as inversion but using names in :n:`@or_and_intropattern_loc`
-   for naming hypotheses. The :n:`@or_and_intropattern_loc` must have the form
-   :n:`[p`:sub:`11` :n:`... p`:sub:`1n` :n:`| ... | p`:sub:`m1` :n:`... p`:sub:`mn` :n:`]`
+   This generally behaves like inversion but using names in :n:`@as_or_and_ipat`
+   for naming hypotheses. The :n:`@as_or_and_ipat` must have the form
+   :n:`[p__11 … p__1n | … | p__m1 … p__mn]`
```

Comment:
This paragraph is long and dense.  I think some of it is repeating the explanation of how `@as_or_and_ipat` works, which should be explain in one place (not here).  Perhaps you can help me figure out what parts are unique to this tactic?

Reply:
Most of it is specific to the `inversion` tactic. And honestly, I don't think that many users are actually using this form because the variables that `inversion` generate are quite unpredictable.